packages/neutrino/test/api_test.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-vars */
```

Comment:
I think we should disable this rule per-line rather than globally for the file, otherwise genuinely unused variables won't be reported.